### DISCUSSION TITLE
Language setting

### DISCUSSION THREAD

**Discussion by LLLgoyour** [ppy/osu#13777] [Q&A] [answered]
Hey guys, I just found that Osu! on mac can't load Chinese. Is there any language package for Chinese or it's still waited to add?

**Comment by smoogipoo** [comment, marked as answer]
Very little has been translated for now. You can help out by following this example PR: https://github.com/ppy/osu/pull/13720

**Reply by LLLgoyour** [reply]
Thanks for the response! I'll find out in that.

**Reply by LLLgoyour** [reply]
Do you still commit the contribution from osu-translate-zh, or the osu Chinese translation organization in mac project?

**Reply by smoogipoo** [reply]
This (osu!lazer) is different from the mac project (do you mean https://osx.ppy.sh ?)

**Reply by LLLgoyour** [reply]
nuh, i mean do you use contribution from https://github.com/osu-translate-zh to do the translation?

**Reply by peppy** [reply]
no. we use only official projects.

**Reply by LLLgoyour** [reply]
alright. is there a direct repo for chinese translation now? or just pull the request to the repo

**Reply by peppy** [reply]
Please wait until the process is setup. You can help here for now https://crowdin.com/project/osu-web. New localisations will appear here in the future as we expand scope.

**Reply by LLLgoyour** [reply]
gotcha
